src/app/pages/information/information.page.scss

```diff
@@ -0,0 +1,12 @@
+ion-slides {
+    width: 100%;
+  }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6012707)